source/MRViewer/MRToolbar.h

```diff
@@ -33,6 +33,8 @@ class Toolbar
     void resetItemsList();
     /// get acces to items
     const MenuItemsList& getItemsList() const { return itemsList_; }
+
+    void setScaling( float scale ) { scaling_ = scale; }
```

Comment:
mb just take it from ribbon menu? `ribbonMenu_->menu_scaling()`

Reply:
i think it same. now this method calls in RibbonMenu::postRescale_